apps/pigment-css-vite-app/src/pages/experiments/index.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { styled } from '@pigment-css/react';
+
+const A = styled('div')({
+  minWidth: 300,
+  minHeight: 100,
+  padding: 10,
+  margin: 10,
+  background: 'orange',
+  color: 'white',
+  variants: [{
+    props: { variant: 'primary' },
+    style: {
+      background: 'blue',
+    },
+  }]
+});
+
+const B = styled(A)({
+  background: 'red',
+  variants: [{
+    props: { variant: 'secondary' },
+    style: {
+      background: 'purple',
+    },
+  }]
+})
+
+export default function Overrides() {
```

Comment:
It's strange, the out put is identical with the one using Emotion

<img width="540" alt="Screenshot 2024-08-13 at 16 53 26" src="https://github.com/user-attachments/assets/3d0c20a6-38b0-47e7-bfdc-386a7ea54c1e">